t_regress/test_reboot_under_load.py

```diff
@@ -0,0 +1,175 @@
+"""
+Test TempestaFW reeboot under load.
+"""
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2017-2024 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+import time
+
+from framework import tester
+from framework.parameterize import parameterize_class
+from helpers import remote, tf_cfg
+from run_config import CONCURRENT_CONNECTIONS, DURATION, REQUESTS_COUNT, THREADS
+
+STATUS_OK = "200"
+
+TEMPESTA_NO_CACHE = {
+    "config": """
+        listen ${tempesta_ip}:443 proto=h2;
+
+        server ${server_ip}:8000;
+        
+        tls_certificate ${tempesta_workdir}/tempesta.crt;
+        tls_certificate_key ${tempesta_workdir}/tempesta.key;
+        tls_match_any_server_name;
+        
+        cache 0;
+        block_action error reply;
+        block_action attack reply;
+    """
+}
+
+TEMPESTA_WITH_CACHE = {
+    "config": """
+        listen ${tempesta_ip}:443 proto=h2;
+
+        server ${server_ip}:8000;
+        
+        tls_certificate ${tempesta_workdir}/tempesta.crt;
+        tls_certificate_key ${tempesta_workdir}/tempesta.key;
+        tls_match_any_server_name;
+        
+        cache 2;
+        cache_fulfill * *;
+
+        block_action error reply;
+        block_action attack reply;
+    """
+}
+
+
+@parameterize_class(
+    [
+        {"name": "tfw_no_cache", "tempesta": TEMPESTA_NO_CACHE},
+        {"name": "tfw_with_cache", "tempesta": TEMPESTA_WITH_CACHE},
+    ]
+)
```

Comment:
You should use `parameterize_class` here
```suggestion
@parameterize_class(
    [
        {"name": "TfwNoCache", "tempesta": TEMPESTA_NO_CACHE, "restart_timeout": 10, "warm_timeout": 0},
        {"name": "TfwWithCache", "tempesta": TEMPESTA_WITH_CACHE, "restart_timeout": 10, "warm_timeout": 0},
        {"name": "NoTimeoutTfwNoCache", "tempesta": TEMPESTA_NO_CACHE, "restart_timeout": 0, "warm_timeout": 5},
        {"name": "NoTimeoutTfwWithCache", "tempesta": TEMPESTA_WITH_CACHE, "restart_timeout": 0, "warm_timeout": 5},
    ]
)
```

Reply:
Fixed ✅